src/containers/CreateCourse/CreateCourse.test.jsx

```diff
@@ -48,36 +48,45 @@ const initialState = {
 
 const middlewares = [thunk];
 const mockStore = configureStore(middlewares);
-let mockOnSubmit;
 let store;
 
 const createWrapper = (state) => {
   store = mockStore(state);
-  mockOnSubmit = jest.fn();
-  return shallow(<CreateCoursePage
-    store={store}
-    createCourse={mockOnSubmit}
-  />);
+
+  return mount((
+    <MemoryRouter>
+      <Provider store={store}>
+        <CreateCoursePage
+          initialValues={courseData}
+        />
+      </Provider>
+    </MemoryRouter>
+  ));
 };
 
 describe('Create Course View', () => {
   it('shows spinner while loading', () => {
     const testState = jsonDeepCopy(initialState);
     testState.publisherUserInfo.isFetching = true;
-    const wrapper = createWrapper(testState).find('CreateCoursePage').dive();
-    expect(wrapper.find('LoadingSpinner').dive()).toHaveLength(1);
+
+    const wrapper = createWrapper(testState).find('CreateCoursePage');
+    expect(wrapper.find('LoadingSpinner')).toHaveLength(1);
   });
 
   it('shows error when fails to retrieve organizations', () => {
     const testState = jsonDeepCopy(initialState);
     const errorMessage = ['organization failure'];
     testState.publisherUserInfo.isFetching = false;
     testState.publisherUserInfo.error = errorMessage;
-    testState.courseOptions.isFetching = false;
-    testState.courseRunOptions.isFetching = false;
-    const wrapper = createWrapper(testState).find('CreateCoursePage').dive();
-    expect(wrapper.find('#create-error')).toHaveLength(1);
-    expect(wrapper.find('#create-error').text()).toEqual(errorMessage[0]);
+    testState.courseOptions.isFetching = true;
```

Comment:
We changed `isFetching` from `false` to `true`. Why?

Reply:
Brought it back to `false`. Thanks.